src/containers/CodesplainAppBar.jsx

```diff
@@ -7,6 +7,7 @@ import {
 } from 'material-ui';
 import { withRouter } from 'react-router';
 import cookie from 'react-cookie';
+import { reduce, concat, sortBy } from 'lodash';
```

Comment:
Check the other files, like [here](https://github.com/maryvilledev/codesplainUI/blob/master/src/util/annotations.js#L1) as an example of how you should import lodash. Even though you're only importing a few functions from the default lodash object, you still have to import the entire thing, so it effectively is the same as before. Instead, it should be:
```js
import reduce from 'lodash/reduce';
/// ...
```